schema/crdb/scim-users-and-groups-lower/up03.sql

```diff
@@ -0,0 +1 @@
+DROP INDEX IF EXISTS lookup_silo_user_by_silo_and_user_name
```

Comment:
Missing semicolon here, but clearly this is not a problem since it works.

Reply:
Fixed in 80f5a5c7